src/components/experiment-tracking/parallel-coordinates/parallel-coordinates.js

```diff
@@ -99,7 +100,7 @@ export const ParallelCoordinates = ({
 
   const handleMouseOverMetric = (e, key) => {
     const runsCount = graph.find((each) => each[0] === key)[1].length;
-    const { x, y, direction } = getTooltipPosition(e);
+    const { x, y, direction } = getTooltipPosition(e, sidebarVisible);
```

Comment:
We use the `Boolean` of `sidebarVisible` to determine if we need to adjust the tooltip's position.